tests/integration_tests/charts/api_tests.py

```diff
@@ -2330,57 +2326,3 @@ def test_update_chart_no_tag_changes(self):
 
         security_manager.add_permission_role(alpha_role, write_tags_perm)
         security_manager.add_permission_role(alpha_role, tag_charts_perm)
-
-    @patch("superset.security.manager.SupersetSecurityManager.has_guest_access")
-    @patch("superset.security.manager.SupersetSecurityManager.is_guest_user")
-    @pytest.mark.usefixtures("load_birth_names_dashboard_with_slices")
-    def test_get_chart_data_as_guest_user(
```

Comment:
I just moved this test to the `/tests/integration_tests/charts/data/api_tests.py` file.